### DISCUSSION TITLE
Thermal Foundation Ores

### DISCUSSION THREAD

**Discussion by BlankSBG** [lukebemish/excavated_variants#194] [Q&A]
im making a modpack for 1.19.2 and im using a couple mods that have duplicate ores, for example mekanism and thermal. through globalresources\excavated_variants\excavated_variants\variants i put in the attached file (github wont let me attach json5) but when i loaded the world none of the thermal ores were put into the system, i looked around my mc installation and noticed the mapping cache in the mod_data folder mentioned some of the mods i have installed, mekanism, create, unearthed, biomesop, and immersive engineering, but no mention of thermal foundation
[thermalfoundation.json](https://github.com/lukebemishprojects/ExcavatedVariants/files/14235901/thermalfoundation.json)

**Comment by lukebemish** [comment]
Could you provide a log please?

**Reply by BlankSBG** [reply]
sure thing, this is me loading in, creating a fresh new world, and quitting the game

[debug.log](https://github.com/lukebemishprojects/ExcavatedVariants/files/14255110/debug.log)
[latest.log](https://github.com/lukebemishprojects/ExcavatedVariants/files/14255112/latest.log)

**Reply by lukebemish** [reply]
Hmm, try removing "deepslate" from the "types" block of each entry in your config?

**Reply by lukebemish** [reply]
Also, do the blocks not exist *at all* (check creative inventory) or do they just not appear in the world? And do you have more than one mod that adds tin/whatever ore?

**Reply by BlankSBG** [reply]
theyre not in the creative inv tab, this is before removing deepslate entries 
![Screenshot 2024-02-12 213840](https://github.com/lukebemishprojects/ExcavatedVariants/assets/159747295/fb213cdc-edb8-4504-ad44-d8d5d13bd8c7)
 yes i have at least two mods that add their own tin or other ore, being thermal, mekanism, EI, but ive gone through each mods configs and disabled the respective duplicate ores but left thermal being the dominant ore distributor because i like how they look.

**Reply by BlankSBG** [reply]
ive just realized the logs i gave mightve been a little inaccurate, im using another computer to host a server for myself for various reasons and ive made a few changes in configs on the server that i didnt make to my client because i thought it "wanst necessary" lol, and the logs i gave were of me generating a world on my client, so some ores generated in that world that otherwise wouldnt.

these logs are from my server generating a new world, and ive omitted deepslate from the types section of every ore. 
[debug.log](https://github.com/lukebemishprojects/ExcavatedVariants/files/14256087/debug.log)
[latest.log](https://github.com/lukebemishprojects/ExcavatedVariants/files/14256088/latest.log)
looking around underground all the ores i wanted to generate generated and the ones i didnt want i didnt find. mekanism, create, and IE ores are recognized by your mod in the creative inv, mekanisms osmium spawns and tin doesnt, as intended, all the thermal ores i want generated but were not varianted

**Reply by lukebemish** [reply]
Do you have an `excavated_variants.zip` file in your `globalresources` folder? Try naming the file you're putting stuff into something different - like, `globalresources\custom\excavated_variants\variants` - and see if that makes a difference.

**Reply by BlankSBG** [reply]
![image](https://github.com/lukebemishprojects/ExcavatedVariants/assets/159747295/9a9d71a5-1187-4eb2-aa02-e9fd9bf07cf8)
this is the folder formatting my client and server use, do you want me to rename the zip?

**Reply by BlankSBG** [reply]
changing the names / unzipping the folders doesnt change anything

**Reply by BlankSBG** [reply]
should i just cut my losses and use different ores that are compatible by default?

**Reply by lukebemish** [reply]
Hmm, this is really strange. It should be working out of the box; I have a couple things to test, but I need to try it locally which i haven't been able to do yet.

**Reply by BlankSBG** [reply]
any updates?
i could use the other ores from the other tech mods i have, but theres some ores in thermal that arent present in the other mods and they would look out of place

**Reply by lukebemish** [reply]
I haven't had a chance to poke this yet as unfortunately I've been rather busy, but I'll try to get to it soon

**Reply by BlankSBG** [reply]
dont stress it, take your time, thank you for your work
